components/alerts/alerts.html

```diff
@@ -46,8 +46,87 @@
                     </ul>
                 </div>
             </div>
+            <div class="doc-content">
+                <div class="content-title">
+                    Alerts
+                </div>
+                <div class="content-subtitle">
+                    An alert is a message that is displayed in a way that attracts the user's attention without interrupting their flow.
+                </div>
+                
+
+                <div class="divider"></div>
+
+                <!-- SIMPLE ALERTS -->
+                <div class="content-head">
+                    Simple Alerts
+                </div>
+
+                <div class="content-subtitle">
+                    To generate a simple alert use the base class <b>alert</b>. There are 5 types of themed alerts - <b>Primary, Secondary, Success, Warning, and Danger</b> alerts.
+                    <br>
```

Comment:
Everything looks good but try using padding bottom or margin bottom utility classes instead of br tag